### ISSUE TITLE
Editor only method called in BaseMeshSync.cs 

### ISSUE THREAD

**Issue by SolarianZ** [unity3d-jp/MeshSync#513]
MeshSync version: 0.10.1-preview

In BaseMeshSync.cs line 1816-1819:

```csharp
            } else if (materials[si] == null) {
                materials[si] = FindDefaultMaterial();
                changed = true;
            }
```

Method `FindDefaultMaterial()` was defined with condition `UNITY_EDITOR`

**Comment by David-Everlasting-Media** [comment]
Getting an error pertaining to this section. 
Attempting to build results in error: 

`Library\PackageCache\com.unity.meshsync@0.10.1-preview\Runtime\Scripts\BaseMeshSync.cs(1817,33): error CS0103: The name 'FindDefaultMaterial' does not exist in the current context`

**Comment by sindharta** [comment]
Thank you for the report. 
Will deploy a fix soon.

**Comment by sindharta** [comment]
Fixed in 0.10.2-preview.
